recipes/weeder/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+TGT="$PREFIX/share/weeder2"
+[ -d "$TGT" ] || mkdir -p "$TGT"
+[ -d "${PREFIX}/bin" ] || mkdir -p "${PREFIX}/bin"
+
+cd "${SRC_DIR}"
+g++ weeder2.cpp -o weeder2 -O3
+cp -r FreqFiles $TGT/
```

Comment:
Thats big. Could you use the post installation hooks to download this DB? here is one small example: https://github.com/bioconda/bioconda-recipes/blob/2d8dcb197920588ddc6c27195fe9a2d41ddc3c1f/recipes/augustus/3.2.2/build.sh#L29